packages/kotti-ui/source/kotti-field-select/components/ActionIcon.vue

```diff
@@ -1,52 +1,65 @@
 <template>
-	<div
-		:class="classes"
-		role="button"
-		@mouseenter="hoverOnClearIcon = true"
-		@mouseleave="hoverOnClearIcon = false"
-	>
-		<i class="yoco" @click.stop="handleClearIfShown" v-text="icon" />
+	<div :class="classes" role="button">
+		<div class="action-icon__container">
+			<i
+				v-if="showClear"
+				class="yoco"
+				@click="handleClear"
+				v-text="Yoco.Icon.CLOSE"
+			/>
+			<i
+				class="yoco"
+				@click.stop.prevent="handleSetIsDropdownOpen"
+				@mousedown.stop.prevent
+				v-text="isDropdownOpen ? Yoco.Icon.CHEVRON_UP : Yoco.Icon.CHEVRON_DOWN"
+			/>
+		</div>
 	</div>
 </template>
 
 <script lang="ts">
 import { Yoco } from '@3yourmind/yoco'
-import { defineComponent, ref, computed } from '@vue/composition-api'
+import { defineComponent, computed } from '@vue/composition-api'
 
 export default defineComponent({
 	name: 'ActionIcon',
 	props: {
 		classes: { required: true, type: Array },
 		handleClear: { required: true, type: Function },
 		isDropdownOpen: { required: true, type: Boolean },
+		isFieldHoveredOrFocused: { required: true, type: Boolean },
 		showClear: { required: true, type: Boolean },
 	},
-	setup(props: {
-		classes: string[]
-		handleClear(): void
-		isDropdownOpen: boolean
-		showClear: boolean
-	}) {
-		const hoverOnClearIcon = ref(false)
-
-		const canClear = computed<boolean>(
-			() => props.showClear && hoverOnClearIcon.value,
+	setup(
+		props: {
+			classes: string[]
+			handleClear(): void
+			isDropdownOpen: boolean
+			isFieldHoveredOrFocused: boolean
+			showClear: boolean
+		},
+		{ emit },
+	) {
+		const showClear = computed<boolean>(
```

Comment:
this is going to break things

I didn't realise there was a prop with the same name when I first reviewed this, I'll fix now